Linear_cell_complex/include/CGAL/Linear_cell_complex_for_combinatorial_map.h

```diff
@@ -205,7 +205,7 @@ namespace CGAL {
         import_from_halfedge_graph(heg, default_point_converter,
                                    origin_to_copy, copy_to_origin);
       }
-
+      using Base::clear;
```

Comment:
@gdamiand Is this OK ? My intention is to fix the helper using `Has_member_clear<>` that doesn't detect LCC::clear() in clear(FaceGraph). 
With this it does, and I think it's OK because the General_map concept contains a clear() function, and a LCC_for_combinatorial_maps refines the concept.

Reply:
@maxGimeno Yes this is ok ! Thanks.